Makefile

```diff
@@ -57,7 +57,7 @@ lint: golint lint-markdown lint-python
 
 golint: | vendor
 	@echo Linting Go files...
-	@gometalinter --vendor --deadline=90s --enable=gofmt --disable=gotype ./...
+	@gometalinter --vendor --deadline=90s --enable=gofmt --disable=gotype --linter='gas:gas -exlude=G104:' ./...
```

Comment:
We want to ensure we have all error return values checked. I'm not sure how the failure that happens in status.go was missed in the previous runs when the errcheck linter would run before though.